README.rst

```diff
@@ -1,8 +1,7 @@
 .. figure:: https://github.com/pybind/pybind11/raw/master/docs/pybind11-logo.png
    :alt: pybind11 logo
 
-pybind11 — Seamless operability between C++11 and Python
-========================================================
+**pybind11 — Seamless operability between C++11 and Python**
```

Comment:
This change looks wrong. You're changing it from "a section" to "emphasised text". Are you sure this is correct?

Reply:
It's currently the only way I can find to get the table of contents working in the PDF. (And, for this one change, the pybind11 title, it happens to look better too, since the image is the actual "heading" - maybe not the other ones).  The problem is that sections in the index.rst are used instead of the table of contents for PDFs. I've played with several other ideas. `.. only::` cannot affect the section structure. A Transformer happens after the structure is parsed.

Since the readme is short and has not TOC of its own, I don't think making these "real" section headers is all that important.

I'm open to other ideas!

Reply:
I expected as much. It's good enough for me.

Reply:
I found a way to keep these using a bit of a hack. :)

Reply:
PS: The LaTeX ToC is known to have issues, see https://groups.google.com/g/sphinx-users/c/_zIypOv2d4g